packages/expo-image/src/utils/resolveHashString.tsx

```diff
@@ -0,0 +1,22 @@
+import { ImageSource } from '../Image.types';
+
+function hashToUri(type: 'blurhash' | 'thumbhash', hash: string): string {
+  const encodedBlurhash = encodeURI(hash).replace(/#/g, '%23').replace(/\?/g, '%3F');
+  return `${type}:/${encodedBlurhash}`;
```

Comment:
Compared to blurhash, is there no way to specify width and height of the image?

Reply:
Yep, in ThumbHash width and height are automatically encoded into the hash when generating. Aspect ratio (approximate) can be read on the native side.